src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/WorkArounds.java

```diff
@@ -194,74 +194,6 @@ public TypeElement searchClass(TypeElement klass, String className) {
         return elementUtils.getTypeElement(className);
     }
 
-    // TODO:  need to re-implement this using j.l.m. correctly!, this has
-    //        implications on testInterface, the note here is that javac's supertype
-    //        does the right thing returning Parameters in scope.
-    /*
-     * Returns the closest superclass (not the superinterface) that contains
-     * a method that is both:
-     *
-     *   - overridden by the specified method, and
-     *   - is not itself a *simple* override
-     *
-     * If no such class can be found, returns null.
-     *
-     * If the specified method belongs to an interface, the only considered
-     * superclass is java.lang.Object no matter how many other interfaces
-     * that interface extends.
-     */
-    public DeclaredType overriddenType(ExecutableElement method) {
-        if (utils.isStatic(method)) {
-            return null;
-        }
-        MethodSymbol sym = (MethodSymbol) method;
-        ClassSymbol origin = (ClassSymbol) sym.owner;
-        for (Type t = javacTypes.supertype(origin.type);
-             t.hasTag(TypeTag.CLASS);
-             t = javacTypes.supertype(t)) {
-            ClassSymbol c = (ClassSymbol) t.tsym;
-            for (Symbol sym2 : c.members().getSymbolsByName(sym.name)) {
-                if (sym.overrides(sym2, origin, javacTypes, true)) {
-                    // Ignore those methods that may be a simple override
-                    // and allow the real API method to be found.
-                    if (utils.isSimpleOverride((MethodSymbol)sym2)) {
-                        continue;
-                    }
-                    assert t.hasTag(TypeTag.CLASS) && !t.isInterface();
-                    return (Type.ClassType) t;
-                }
-            }
-        }
-        return null;
-    }
-
-    // TODO: the method jx.l.m.Elements::overrides does not check
-    // the return type, see JDK-8174840 until that is resolved,
-    // use a  copy of the same method, with a return type check.
-
-    // Note: the rider.overrides call in this method *must* be consistent
-    // with the call in overrideType(....), the method above.
-    public boolean overrides(ExecutableElement e1, ExecutableElement e2, TypeElement cls) {
-        MethodSymbol rider = (MethodSymbol)e1;
-        MethodSymbol ridee = (MethodSymbol)e2;
-        ClassSymbol origin = (ClassSymbol)cls;
-
-        return rider.name == ridee.name &&
-
-               // not reflexive as per JLS
-               rider != ridee &&
-
-               // we don't care if ridee is static, though that wouldn't
-               // compile
-               !rider.isStatic() &&
-
-               // Symbol.overrides assumes the following
-               ridee.isMemberOf(origin, javacTypes) &&
-
-               // check access, signatures and check return types
-               rider.overrides(ridee, origin, javacTypes, true);
-    }
-
     // TODO: jx.l.m ?
```

Comment:
Not edited, but I note that Language Model and Compiler API have a very distant relationship :-(